components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -177,6 +185,16 @@ void get_rain_dsd2(GetRainDsd2Data& d)
 			    d.rhorime_c,      &d.th,    &d.qv,    &d.qitot, &d.nitot, &d.qirim,
 			    &d.birim,         &d.qc,    &d.nc,    &d.qr, &d.nr);
   }
+
+void  update_prognostic_liquid(P3UpdatePrognosticLiqData& d){
+  p3_init(true);
+  update_prognostic_liquid_c(d.qcacc, d.ncacc, d.qcaut, d.ncautc, d.qcnuc, d.ncautr,
+			      d.ncslf, d. qrevp, d.nrevp, d.nrslf , d.log_predictNc,
+			      d.inv_rho, d.exner, d.xxlv, d.dt, &d.th, &d.qv,
+			      &d.qc, &d.nc, &d.qr, &d.nr);
+  }
+
```

Comment:
And again here. We've generally been following the pattern of a single blank line separating definitions.